zgw/notificaties-api/src/main/kotlin/com/ritense/notificatiesapi/client/NotificatiesApiClient.kt

```diff
@@ -58,11 +58,16 @@ class NotificatiesApiClient(
             .awaitBodilessEntity()
     }
 
-    internal suspend fun createKanaal(authentication: NotificatiesApiAuthentication, baseUrl: URI, kanaal: Kanaal): Kanaal {
+    internal suspend fun createKanaal(
+        authentication: NotificatiesApiAuthentication,
+        baseUrl: URI,
+        kanaal: Kanaal
+    ): Kanaal {
         return buildNotificatiesWebClient(authentication, baseUrl)
             .post()
             .uri("kanaal")
             .contentType(MediaType.APPLICATION_JSON)
+            .headers { it.setAll(mapOf("header1" to "value1", "header2" to "value2")) }
```

Comment:
This shouldn't be there

Reply:
whoops, this was a mistake and was used to make a point earlier haha, sorry! 🙏